### ISSUE TITLE
Keyword arguments are reported as `Unbound`

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#10811]
In the following code sample, the keywords used in the last function call are reported as potentially unbound.

```python
import json

def foo() -> tuple[str, str]: ...

try:
    a, b = foo()
    c = "".join(a)
    e = json.loads("")
except:
    e = {}

f = e.get("", None)

g = dict(f=f, c=c) # f is reported as Unbound
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.405
